go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/replicatedhq/kots
 
-go 1.20
+go 1.21.0
+
+toolchain go1.21.3
```

Comment:
this appears to be required as part of importing embedded-cluster-operator

Reply:
would it make it an easier import if there was a separate module for the kinds/apis as we have for kotskinds and kurlkinds?

Reply:
Might be something to consider certainly